releases.yaml

```diff
@@ -1,17 +1,19 @@
 latest:
-  slug: ManticMinotaur
-  name: "Mantic Minotaur"
-  short_version: "23.10"
-  full_version: "23.10"
-  desktop_version: "23.10.1" # for 23.10 release only, because Desktop image for 23.10 release has version 23.10.1, while as all other images have version 23.10
-  core_version: "24"
-  release_date: "October 2023"
-  eol: "July 2024"
+  slug: OracularOriole
+  name: "Oracular Oriole"
+  short_version: "24.10"
+  full_version: "24.10"
+  desktop_version: "TBA"
+  core_version: "TBA"
+  release_date: "October 2024"
+  eol: "July 2025"
   past_eol_date: false
-  release_notes_url: "https://discourse.ubuntu.com/t/mantic-minotaur-release-notes/35534"
-  iso_download_size: "4.5GB"
-  server_iso_size: "2.5GB"
-  rasp_pi_core_24_size: "314MB"
+  release_notes_url: "TBA"
```

Comment:
@lorumic you can put this link here as it will not change until the release: https://discourse.ubuntu.com/t/oracular-oriole-release-notes/44878

Reply:
Done: https://github.com/canonical/ubuntu.com/pull/14290/commits/1a8db636da12745c1f83c0bed607c6e9adead187